### ISSUE TITLE
IOS, whole string is replaced by emoji or .com

### ISSUE THREAD

**Issue by serrg1994** [NativeScript/NativeScript#10108]
### Issue Description

Noticed starting from 8.2.0 version strange behaviour for TextField on iOS. For example if keyboard type is specified as url and user is typing url and then press **.com** whole string will be replaced by **.com**. The same behaviour I see for emojis 

https://user-images.githubusercontent.com/9201885/204058660-8fd632a4-2617-47dd-8306-97a791315aff.mov

### Environment

```yaml
OS: macOS 13.0.1
CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
Shell: /bin/zsh
node: 14.19.0
npm: 8.5.0
nativescript: 8.3.3

# android
java: 18.0.1
ndk: Not Found
apis: Not Found
build_tools: Not Found
system_images: Not Found

# ios
xcode: 14.1/14B47b
cocoapods: 1.11.3
python: 2.7.18
python3: 3.9.6
ruby: 2.6.10
platforms: 
  - DriverKit 22.1
  - iOS 16.1
  - macOS 13.0
  - tvOS 16.1
  - watchOS 9.1
```

### Dependencies

```json
"dependencies": {
  "@nativescript/core": "8.3.6",
  "@nativescript/theme": "~3.0.2",
  "nativescript-vue": "~2.9.0"
},
"devDependencies": {
  "@nativescript/ios": "8.3.3",
  "@nativescript/webpack": "~5.0.9",
  "nativescript-vue-template-compiler": "~2.9.0"
}
```

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by serrg1994** [comment]
BTW if keyboard type is url should not be first letter in capital by default ?

**Comment by serrg1994** [comment]
Did anybody check if TextField from UIKit has identical issue with autofill and it requires similar fixes as was done in here https://github.com/NativeScript/NativeScript/pull/9555 ?

**Comment by miladfm** [comment]
Is there any progress on this issue?
This bug is also available in `@nativescript/core`  8.4.2

**Comment by NathanWalker** [comment]
@miladfm @serrg1994 this will go out with 8.4.3 today from ref'd pr.
